cluster/src/main/java/com/hartwig/pipeline/calling/structural/gridss/command/CollectGridssMetrics.java

```diff
@@ -32,10 +33,8 @@ public String arguments() {
                         .asBash();
     }
 
-    // This GRIDSS command uses the "-o" argument to mean the fully-qualified basename for the output files. It will
-    // generate filenames for each of the requested metrics based upon that starting point.
     public String outputBaseFilename() {
-        return VmDirectories.outputFile(format("%s_metrics", GridssCommon.basenameNoExtensions(inputBam)));
+        return format("%s/%s", workingDirectory, new File(inputBam).getName());
```

Comment:
do we need to use the input bam or can we just use the sample name for this file? I find it a little awkward some files are named "CPCT12345678T.sorted.something", would be nice to drop the sorted.

Reply:
Think we might be able to do away with it altogether when we make the whole thing more in the functional style.